cirq/protocols/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""  Protocols and methods for quantum channels, unitary and mixture """
```

Comment:
nit: Please make the comment syntactically consistent with other docstrings in cirq. See [example](https://github.com/quantumlib/Cirq/blob/master/cirq/sim/__init__.py). In particular, we don't use a leading space and we do end sentences in the docstrings with a dot.